docs/releases/3.1.txt

```diff
@@ -597,6 +597,10 @@ Miscellaneous
 * The admin CSS classes ``row1`` and ``row2`` are removed in favor of
   ``:nth-child(odd)`` and ``:nth-child(even)`` pseudo-classes.
 
+* The :func:`~django.contrib.auth.hashers.make_password` now requires its
```

Comment:
`make_password()` docs should also be updated (with `versionchanged` annotation).